src/components/atoms/specialMealCard/SpecialMealCard.stories.tsx

```diff
@@ -9,11 +10,25 @@ export const Default = {
   },
 };
 
-const specialMealCard = {
+const specialMealCardConfig = {
   title: "Atoms/SpecialMealCard",
   component: SpecialMealCard,
   tags: ["autodocs"],
   excludeStories: /.*Data$/,
+  decorators: [
+    (Story: FC) => (
+      <div
+        style={{
+          backgroundColor: "#fff8ee",
+          height: "180px",
+          width: "180px",
+          borderRadius: "20px",
```

Comment:
Why do you put height, width, borderRadius?

Reply:
I put them to look exactly the same as in figma. I put them just in stories; isn't it just presentation purposes.